Samples/AppLifecycle/StateNotifications/cpp-win32-unpackaged/CppWinMainState/CppWinMainState.cpp

```diff
@@ -0,0 +1,426 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "framework.h"
```

Comment:
Do I need to do something extra to run the sample? Just building didn't work for me.
_D:\a\_work\1\s\dev\WindowsAppRuntime_BootstrapDLL\MddBootstrap.cpp(339)\Microsoft.WindowsAppRuntime.Bootstrap.dll!00007FF82B2327BF: (caller: 00007FF82B231307) Exception(1) tid(d020) 80070491 There was no match for the specified key in the index.
    Msg:[AppExtension.Name=microsoft.winappruntime.ddlm-1.0-x6-p1, Major=1, Minor=0, Tag=preview1, MinVersion=0.0.0.0] 
Exception thrown at 0x00007FF85745466C in CppWinMainState.exe: Microsoft C++ exception: wil::ResultException at memory location 0x000000B5258FCC30.
Exception thrown at 0x00007FF85745466C in CppWinMainState.exe: Microsoft C++ exception: [rethrow] at memory location 0x000000000000000_

Reply:
Check you have preview1 installed.